notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    t_class = (
</pre><p>myabe <code>g_class</code> (for "grouped") or <code>by_class</code> ? I am not sure as a reader what the "t_" is supposed to connote.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>"titanic_class" was what I had in mind, but <code>by_class</code> sounds good as well. </p>